tools/matchms/matchms_wrapper.py

```diff
@@ -17,21 +17,28 @@ def main(argv):
     parser.add_argument("-f", dest="default_filters", action='store_true', help="Apply default filters")
     parser.add_argument("-n", dest="normalize_intensities", action='store_true', help="Normalize intensities.")
     parser.add_argument("-s", dest="symmetric", action='store_true', help="Computation is symmetric.")
-    parser.add_argument("--ref", dest="references_filename", type=str, help="Path to reference MSP library.")
+    parser.add_argument("--ref", dest="references_filename", type=str, help="Path to reference spectra library.")
     parser.add_argument("queries_filename", type=str, help="Path to query spectra.")
     parser.add_argument("similarity_metric", type=str, help='Metric to use for matching.')
     parser.add_argument("tolerance", type=float, help="Tolerance to use for peak matching.")
     parser.add_argument("mz_power", type=float, help="The power to raise mz to in the cosine function.")
     parser.add_argument("intensity_power", type=float, help="The power to raise intensity to in the cosine function.")
-    parser.add_argument("output_filename_scores", type=str, help="Path where to store the output .csv scores.")
-    parser.add_argument("output_filename_matches", type=str, help="Path where to store the output .csv matches.")
+    parser.add_argument("output_filename_scores", type=str, help="Path where to store the output .tsv scores.")
+    parser.add_argument("output_filename_matches", type=str, help="Path where to store the output .tsv matches.")
     args = parser.parse_args()
 
-    queries_spectra = list(load_from_msp(args.queries_filename))
+    try:
+        queries_spectra = list(load_from_msp(args.queries_filename))
+    except ValueError:
+        queries_spectra = list(load_from_mgf(args.queries_filename))

```

Comment:
this doesn't really look like a good solution! Maybe we can add the file type as a flag to the wrapper and then call the respective function based on the file type? This could also be done via a dictionary, mapping the extension to the loader function.

Reply:
I agree, although I don't know how do it without asking user to explicitly choose the file type and then the file itself, which seems laborious to me.

I think the file extension cannot be used, because Galaxy is using `.dat` files for all data internally.

Reply:
> It can be used, using the ${var.ext} tag, see https://github.com/RECETOX/galaxytools/blob/e4cdacabd9d58524d3271e005644b8f2d56d83fc/tools/riassigner/riassigner.xml#L12

Reply:
That is a cool feature, I didn't know about it. Thanks! Fixed in 0c7ba48.